src/site/topic2-cpp.rst

```diff
@@ -0,0 +1,646 @@
+******************************************
+Topic #1 Aside --- C++ vs. Java vs. Python
+******************************************
+
+* For these, I will only go over the C++ details at a high level
+* Getting into the nitty gritty of C++ is not the purpose of this aside
+
+**Python**
+
+.. code-block:: python
+    :linenos:
+
+    # Python --- hello world
+    print("Hello, world!")
+
+
+**Java**
+
+.. code-block:: java
+    :linenos:
+
+    // Java --- hello world
+    public class SomeClass {
+        public static void main(String[] args){
+            System.out.println("Hello, world!");
+        }
+    }
+
+
+**C++**
+
+.. code-block:: cpp
+    :linenos:
+
+    #include <iostream>
+
+    // Cpp --- hello world
+    int main() {
```

Comment:
I didn't use em' in Python, but we will use them with Java. I'm a tad conflicted. I think leaving it as is is easiest, RE 

>Things are kept brief because (a) too much info will just scare students off, (b) the extra details may just confuse, and (c) I don't want to go into more C++ details.